tests/dummy/app/templates/application.hbs

```diff
@@ -1,3 +1,3 @@
-{{double-toggle-button}}
+<DoubleToggleButton />
 
-{{simple-list data-test-simple-list-wrapper=true}}
+<SimpleList data-test-simple-list-wrapper />
```

Comment:
Change to angle-bracket invocations was required to support upgrading ember-test-selectors to latest; in latest, the curly-invocation doesn't pick up the data-test attrs.